docs/how-to-use/search/advanced-search-user-docs.md

```diff
@@ -0,0 +1,65 @@
+# Advanced Search Component
+
+## Introduction
+
+The `AdvancedSearchComponent` allows for the creation of complex queries using a graphical widget.
+The widget's contents are then turned into a string representing a Gravsearch (SPAQRL) query to be sent to DSP-API.
+
+A query consists of the following elements:
+- ontology (data model) selection
+- selection of a resource class belonging to the selected ontology (optional)
+- specification of properties, comparison operators, and values (optional).
+
+Although selection of a resource or a property ore both are optional, either a resource class has to be selected or at least one property has to be specified,
```

Comment:
ore  -> or (unless we're mining now 😜)

Reply:
fixed in https://github.com/dasch-swiss/dsp-ui-lib/pull/295/commits/8d3c3f20a65273bfdbca05eb12bdd410426edd19